Lib9c/BlockHelper.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Libplanet.Blockchain;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Nekoyume.Action;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+
+namespace Nekoyume
+{
+    public static class BlockHelper
+    {
+        public static Block<PolymorphicAction<ActionBase>> MineGenesisBlock(
+            IDictionary<string, string> tableSheets,
+            GoldDistribution[] goldDistributions)
+        {
+            if (!tableSheets.TryGetValue(nameof(GameConfigSheet), out var csv))
+            {
+                throw new KeyNotFoundException(nameof(GameConfigSheet));
+            }
+            var gameConfigState = new GameConfigState(csv);
+            var redeemCodeListSheet = new RedeemCodeListSheet();
+            redeemCodeListSheet.Set(tableSheets[nameof(RedeemCodeListSheet)]);
+
+
+            // FIXME 메인넷때는 따로 지정해야합니다.
```

Comment:
If this comment is useful, then how about translating to English?

Reply:
Fix at e217c27